creator/data_reviews/models.py

```diff
@@ -0,0 +1,152 @@
+import uuid
+from django.db import models
+from django.contrib.auth import get_user_model
+from django_fsm import FSMField, transition
+
+from creator.fields import kf_id_generator
+from creator.studies.models import Study
+from creator.files.models import File
+
+User = get_user_model()
+
+
+def data_review_id():
+    return kf_id_generator("DR")
+
+
+class State(object):
+    NOT_STARTED = "not_started"
+    IN_REVIEW = "reviewing"
+    WAITING = "waiting"
+    COMPLETED = "completed"
+    CLOSED = "closed"
+
+
+class DataReview(models.Model):
+    """
+    The iterative process between a data contributor and the Data Resource
+    Center where study data is continually supplied, validated, and refined
+    to meet quality standards.
+    """
+
+    class Meta:
+        permissions = [
+            ("list_all_datareview", "Show all data_reviews"),
```

Comment:
do you need to add `add_my_study_datareview`?

Reply:
Yea eventually, but right now only the Administrators group can do stuff with data reviews. Which other groups should be able to do stuff for data reviews and which CRUD actions should they be able to do?